README.md

```diff
@@ -192,17 +192,35 @@ vars:
     sessionization_trailing_window: number_of_hours # ex: 12
 ```
 
+### Changing the Build Schema
+By default this package will build the Mixpanel staging models within a schema titled (<target_schema> + `_stg_mixpanel`) and Mixpanel final models within a schema titled (<target_schema> + `mixpanel`) in your target database. If this is not where you would like your modeled Mixpanel data to be written to, add the following configuration to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+models:
+    mixpanel:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+      staging:
+        +schema: my_new_schema_name # leave blank for just the target_schema
+```
+
 ## Contributions
 Additional contributions to this package are very welcome! Please create issues
 or open PRs against `master`. Check out 
 [this post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) 
 on the best workflow for contributing to a package.
 
+## Database Support
+This package has been tested on BigQuery, Snowflake and Redshift. 
```

Comment:
If this ends up working for the customer we should update Postgres to be included.